src/app/app.component.ts

```diff
@@ -22,7 +27,7 @@ export class AppComponent implements OnInit {
     this.checkAccount();
 
     // event listeners for authentication status
-    this.broadcastService.subscribe('msal:loginSuccess', (success) => {
+    this.subscription1 = this.broadcastService.subscribe('msal:loginSuccess', (success) => {
```

Comment:
I would call these `loginSuccessSubscription`, loginFailureSubscription`, respetively. And they can be local variables, dont need to be added as instance properties.

Reply:
done!